docs/output.md

```diff
@@ -342,6 +342,120 @@ The saved files are the _good_ files, passing the `dust` or `entropy` filter tre
 
 **Note:** To save output files, set the `--metagenomics_complexity_savefastq` flag
 
+### Metagenomics Profiling
+
+#### MALT
+
+[MALT](https://software-ab.cs.uni-tuebingen.de/download/malt) is a fast replacement for BLASTX, BLASTP and BLASTN, and provides both local and semi-global alignment capabilities.
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `metagenomics_screening/profiling/malt/`
+  - `<sample_id>.rma6`: binary file containing all alignments and taxonomic information of hits that can be loaded into the [MEGAN6](https://uni-tuebingen.de/fakultaeten/mathematisch-naturwissenschaftliche-fakultaet/fachbereiche/informatik/lehrstuehle/algorithms-in-bioinformatics/software/megan6/) interactive viewer
+  - `<sample_id>.blastn.sam`: sparse SAM file containing alignments of each hit (if `--metagenomics_malt_savereads`)
+  - `*.log`: LOG file containing runtime log of MALT. NOTE: If you are running parallel malt runs with `--metagenomics_malt_group_size` set above 0, your log files will be labelled with the name of _one_ of the input files run for each of the parallel executions.
+
+</details>
+
+MALT is a metagenomic aligner (equivalent to BLAST, but much faster). It produces direct alignments of sequencing reads in a reference genome. It is often used for metagenomic profiling or pathogen screening, and specifically in nf-core/eager, of off-target reads from genome mapping.
+
+You will receive output for each library. This means that if you use TSV input and have one library sequenced over multiple lanes and sequencing types, these are merged and you will get mapping statistics of all lanes and sequencing configurations in one value.
+
+The main output of MALT is the `.rma6` file format, which can be only loaded into MEGAN and it's related tools.
+
+You will only receive the `.sam` files if you supply `--metagenomics_malt_savereads` parameters to the pipeline.
```

Comment:
As you mention taxpasta in Kraken2 section, maybe you want to to mention this here too?

Reply:
done